scripts/qa_crowbarsetup.sh

```diff
@@ -3296,6 +3297,13 @@ function deploy_single_proposal()
                 return
             fi
             ;;
+        murano)
+            [[ -n "$want_murano" ]] || return
```

Comment:
if you use `[[ ]]` then you don't need `-n` and quotes
So easier and shorter:
`[[ $want_murano ]] || return`